Content.Server/Drone/DroneSystem.cs

```diff
@@ -145,7 +158,7 @@ private bool NonDronesInRange(EntityUid uid, DroneComponent component)
             {
                 if (HasComp<MindComponent>(entity) && !HasComp<DroneComponent>(entity) && !HasComp<GhostComponent>(entity))
                 {
-                    _popupSystem.PopupEntity(Loc.GetString("drone-too-close"), uid, Filter.Entities(uid));
+                    _popupSystem.PopupEntity(Loc.GetString("drone-too-close", ("being", entity)), uid, Filter.Entities(uid));
                     return true;
```

Comment:
Because interactions are now predicted, this needs to have a `if (IGameTiming.IsFirstTimePredicted)` check before generating popups to avoid generating a bunch of them

Reply:
Fixed